### PR TITLE
Simplify build-manifest

### PR DESCRIPTION
**Pull request by pvanheus**
* Simply flatten all the `info.yml` from the scheme directories into a list of schemes
* Read the header from a file and do away with `organisms,yml`
* Allow the schema directory to be specified using `PRIMER_SCHEMA_DIR` environment variable

### CODE REVIEW

**Comment by bede** [comment]
Thanks! I'm broadly happy. This partially restores some functionality I removed a month or two ago for reading env vars. If this unblocks you I can merge now, otherwise I may add some trivial changes before merging. Thoughts?

File: src/primaschema/lib.py [hunk 1]
```diff
@@ -394,36 +394,31 @@ def build_recursive(
...
 
+def scheme_key(scheme: dict) -> str:
+    family = scheme.get("family", "")
+    version = scheme.get("version", "")
+    return '/'.join([scheme['organism'], family, scheme['name'], version])
+
+
 def build_manifest(root_dir: Path, out_dir: Path = Path()):
     """Build manifest of schemes inside the specified directory"""
-    manifest_field_subset = [
```
Comment (bede) [inline src/primaschema/lib.py:399]: Do you want to all the fields to be carried over to the manifest? I was favouring just a subset. We don't need 10 copies of schema_version (or perhaps we do?)

Comment (pvanheus) [reply src/primaschema/lib.py:399]: We can probably killed `schema_version` (presuming everything is kept up to date) but the more we filter the more we need to pay attention to keeping this up to date with the evolving schema.

Comment (bede) [reply src/primaschema/lib.py:399]: Ok, so maybe we exclude rather than keep fields?

Comment (bede) [reply src/primaschema/lib.py:399]: (a handful of annoying fields like schema version)

Comment (bede) [reply src/primaschema/lib.py:399]: I'll tinker add some commits to this shortly if that's ok

Comment (pvanheus) [reply src/primaschema/lib.py:399]: Addressed in the latest commit

**Pull request merged**